Assingment2.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assingment2;
+import java.util.Scanner;
+
+/**
+ This is Assignment 2.  This program provides a menu with 5 options. It 
+ includes an option to run our first assignment and an option to 
+ run assignment 1 and quit. 
+ */
```

Comment:
I would personally delete the comments that are automatically pre-loaded in. Just for the sake of cleanliness. But I'm picky like that lol.